about_atoms.exs

```diff
@@ -32,14 +36,14 @@ defmodule AboutAtoms do
   end
 
   think "It is surprising to find out that booleans are atoms" do
-    assert_? Koans.Check.atom?(true)
-    assert_? Koans.Check.atom?(false)
+    assert_? is_atom(true)
```

Comment:
Nice - I like that we now expose the user to the underlying function so they can learn it